dao.py

```diff
@@ -22,16 +22,22 @@ def _get_by_params(self, collection, selector):
 
     def _save_one(self, collection, document):
         res = self.__db[collection].insert_one(document)
-        if res.acknowledged:
-            return str(res.inserted_id)
-        return None
+        return str(res.inserted_id) if res.acknowledged else None
```

Comment:
if acknowledged is false does the inserted_id exist ?

Reply:
According to the PyMongo docs, no